test-scripts/local-is-deployment/README.md

```diff
@@ -0,0 +1,23 @@
+## Executing local Identity Server Deployment
+
+### Prerequisites
+1. Inorder to execute the Testgrid executable you should have a mysql setup running locally with username/password, root/root
```

Comment:
* In order to execute the Testgrid executable you should have a mysql server running locally with username/password as root/root

Reply:
Having root/root hard-coded is actually major issue in our code. Can you raise a ticket for this?

Reply:
The database should exist as well, right? What's the db name?